.travis.yml

```diff
@@ -1,8 +1,8 @@
-sudo: required
-language: bash
-
-services:
-  - docker
+language: go
```

Comment:
the current setup is setup to do multi languages - its doing go w/o travis knowing it.  So we should be able to add ruby for the CF side of the project as well.